TMS.Net07.Homework.GeometricFigures/Program.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Text.RegularExpressions;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.Net07.Homework.GeometricFigures
+{
+    class Program
+    {// реализовать валидацию ввода для координат
+        static void Main(string[] args)
+        {
+            int resultToConvertFigureNumber = 0;
+            string pattern = @"drow\d$";
+
+            while (true) {
+                Console.WriteLine($"Choose a shape: {Environment.NewLine}" +
+                    $"1) Rectangle{Environment.NewLine}" +
+                    $"2) Sqaure{Environment.NewLine}" +
+                    $"3) Triangle{Environment.NewLine}" +
+                    $"4) Hexagon{Environment.NewLine}" +
+                    $"5) Circle{Environment.NewLine}" +
+                    $"Enter exit if you want close the program");
+
+                var figureNumber = Console.ReadLine().ToLower();
+                figureNumber = figureNumber.Replace(" ", string.Empty);
+                Console.Clear();
+                if (figureNumber == "exit")
+                {
+                    return;
+                }
+                if (Regex.IsMatch(figureNumber, pattern))
+                {
+                    figureNumber = figureNumber.Remove(0, 4);
+                    resultToConvertFigureNumber = int.Parse(figureNumber);
+                }
+                
+
+                var trueorfalse = int.TryParse(figureNumber, out var isfigurenumber);
+                if (trueorfalse)
+                {
+                    bool firstResultToConvertForFirstPoint;
+                    bool secindResultToConvertForFirstPoint;
+                    bool firstResultToConvertForSecondPoint;
+                    bool secindResultToConvertForSecondPoint;
+                    bool thirdResultToConvertForFirstPoint;
+                    bool thirdResultToConvertForSecondPoint;
+
+                    int FirstCoordinateForFirstPoint;
+                    int SecondCoordinateForFirstPoint;
+                    int FirstCoordinateForSecondPoint;
+                    int SecondCoordinateForSecondPoint;
+                    int ThirdCoordinateForFirstPoint;
+                    int ThirdCoordinateForSecondPoint;
```

Comment:
What if a number of coordinates is greater than 3? 4? 5? 
Suggest using an array for the scenario when the number of values can vary